docs/ide-usage.md

```diff
@@ -86,6 +86,12 @@ The following plugins are needed:
 Supported IntelliJ versions: IntelliJ IDEA Ultimate Edition (required to work with Google App Engine).
 You can sign up for the free [JetBrains student license](https://www.jetbrains.com/student/) if you are a student registered in an educational institution.
 
+**NOTE**
+> If you are migrating from Eclipse, you should delete `.project`, `.classpath`, `.launches`,
+> `src/main/webapp/WEB-INF/classes`, `src/main/webapp/WEB-INF/lib` and `src/main/webapp/WEB-INF/appengine-generated`.\
+> You can also copy `local_db.bin` from `src/main/webapp/WEB-INF/appengine-generated` and copy it to the appropriate
```

Comment:
@LiHaoTan has included instructions how to fix this here: https://github.com/TEAMMATES/teammates/pull/7354/files#diff-1bf07101845de6da98f1c948b82c5ce0R84